### PR TITLE
make: Add defines into the xsdk iio project

### PR DESCRIPTION
**Pull request by cristiantpop**
This will be used to extract variable values and define them into the sdk
project, allowing us to create a functional iio sdk project.

This commit has to be approved first: 
https://github.com/analogdevicesinc/libtinyiiod/pull/27

### CODE REVIEW

**Comment by mchindri** [comment]
You can define the needed variables in the linux makefile  and export them like:
https://github.com/analogdevicesinc/no-OS/blob/a0b72d1bf5674a9ca2c5b05d44452f9dda4ea749/tools/scripts/aducm.mk#L7

And then you just have change in tinyiiod from:
TINYIIOD_VERSION_MAJOR = 0	
TINYIIOD_VERSION_MINOR = 1
to 
TINYIIOD_VERSION_MAJOR ?= 0	
TINYIIOD_VERSION_MINOR ?= 1

**Comment by mchindri** [comment]
A better approach will be to link tinyiiod library from command line:
With a command like:
configapp -app test -set compiler-misc {-L"path_to_tinyiiod" -ltinyiiod}

See xilinx command line documentation: 
http://127.0.0.1:37961/help/index.jsp?topic=%2Fcom.xilinx.sdk.docs.user%2Fxsct%2Fuse_cases%2Fxsct_modify_bsp_settings.html
and 
http://127.0.0.1:37961/help/index.jsp?topic=%2Fcom.xilinx.sdk.docs.user%2Fxsct%2Fuse_cases%2Fxsct_modify_bsp_settings.html

**Comment by scuciurean** [comment]
> A better approach will be to link tinyiiod library from command line:
> With a command like:
> configapp -app test -set compiler-misc {-L"path_to_tinyiiod" -ltinyiiod}
> 
> See xilinx command line documentation:
> http://127.0.0.1:37961/help/index.jsp?topic=%2Fcom.xilinx.sdk.docs.user%2Fxsct%2Fuse_cases%2Fxsct_modify_bsp_settings.html
> and
> http://127.0.0.1:37961/help/index.jsp?topic=%2Fcom.xilinx.sdk.docs.user%2Fxsct%2Fuse_cases%2Fxsct_modify_bsp_settings.html

https://github.com/analogdevicesinc/no-OS/blob/master/tools/scripts/linux.mk#L395

**Comment by mchindri** [comment]
I updated my comment with online links.

File: tools/scripts/linux.mk [hunk 1]
```diff
@@ -440,10 +444,12 @@ xilinx-read-hdf:
 
 .SILENT:xilinx-bsp
 xilinx-bsp:
+	echo $(IIO_DEFS)
```
Comment (scuciurean) [inline tools/scripts/linux.mk:447]: I can't see from where IIO_DEFS is comming

**Comment by cristiantpop** [comment]
V2:
-Add libtinyiiod to the project

**Pull request merged**